docker_run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+###  SET HOST FILE PATHS HERE. All paths must be absolute!  ###
+HOST_CFG=$(pwd)/rabbitmq-worker.conf
+HOST_LOG=$(pwd)/rabbitmq-worker.log
+HOST_ERR=$(pwd)/rabbitmq-worker.err
```

Comment:
Ok, I'll look into it.  It might be as simple as specifying /dev/stdout and /dev/stderr as the log and error files in the config file.